webapp/.eslintrc

```diff
@@ -5,34 +5,12 @@
     "node": true
   },
   "plugins": ["compat"],
-  "globals": {
-    "angular": true,
-    "Tour": true
```

Comment:
"angular" global not supported in new framework version. 

"Tour" global is used in places: 
   1. `bootstrap-tour-standalone.js` Currently not covered in the lint evaluation, it'd need several fixes for lint issues, example: double quote to single quote strings. Let me know if to include the file, I think this original code from a lib, so I think it's fine to exclude.
   2. `main.ts` accessed via window object: `window.Tour`. This is fine, no need rule.
   3. `tour.service.ts` accessed via window object. This is fine, no need rule.